### ISSUE TITLE
Question - Fusion360 STL File Access

### ISSUE THREAD

**Issue by danlohman** [Sky360-Repository/sky360#15]
I'm trying to download current STL files for each hardware component as well as groupings of assembled components.  On some of the groupings I get the error that "This assembly uses files in other projects. To share this assembly, you need Editor access to all those projects."  In the attached example it looks like the problem is that I need access to some sub-components used in the Head v1.7 component. I just requested access but wanted to post this as well.
![image](https://github.com/Sky360-Repository/sky360/assets/5044335/09aef93d-efbb-4393-98c3-864405ed2cb2)
![image](https://github.com/Sky360-Repository/sky360/assets/5044335/f2d3bfd2-6728-4390-ac2c-451665497a81)
![image](https://github.com/Sky360-Repository/sky360/assets/5044335/8123c0b4-c866-462f-ae98-354e718f7188)

**Comment by realsnick** [comment]
@RichardGHopf can you help with access?

**Comment by realsnick** [comment]
btw, @danlohman @RichardGHopf @fcbarros will it be valuable to move the STL files to git? one version control? google folder isn't really for version control and hard to follow changes. Further, now, github support 3d file rendering even in markup

https://docs.github.com/en/repositories/working-with-files/using-files/working-with-non-code-files#

**Comment by danlohman** [comment]
I agree that it would be a great idea to maintain a current version of the STL file (and any others that are needed) for each hardware component in Github.  It may make sense to maintain the main website (sky360.org) code in Github as well if you want to reduce the number of places where update need to be made.

**Comment by Sky360Admin** [comment]
yes, we can host STLs on Git.

hosting them on Drive was just a means of provision for downloads, not for version control. this happens in A360, where all CAD files are hosted.
@danlohman you need access to A360?

the request for access to Joe Webers private folder is, hmmm, a bit problematic. but his sources are all in public folders.
@danlohman dm me what you need.

**Comment by realsnick** [comment]
i'll take on moving the STL files to the git repo

**Comment by Sky360Admin** [comment]
just for everyone to know - STL files are meshes, only useful for 3D printing, not for further modelling. that's why you find here only 3D printable STLs:
https://drive.google.com/drive/folders/1neKtp0gZ_70_DP4wMYcA_6emzVf_1uGb

**Comment by danlohman** [comment]
I was interested in the STL files per hardware component because that file format is usable in 3D web renderings.  If we have already done the hard work to design/build/pull them together, it opens up the ability of showing parts of stations in 3D in tutorials, guides, interactive demos, within the GUI, and much more.  I'd love to visualize how a station is assembled in 3D with the parts coming together in a 3D animation. Its also a great way to get someone comfortable with design and parts before buying.  In addition, It opens up viewing 3D parts outside of programs with limited access and steep learning curves by allowing for simply viewing on a webpage.  Just a few thoughts behind why I asked.  I realize I did not give any explanation for my request.

**Comment by realsnick** [comment]
@RichardGHopf can you help with locations for what @danlohman  is asking for , i'll move it all to STL.

**Comment by Sky360Admin** [comment]
Dan, please go away from STLs - they are huge single meshes.
every renderer would suffer calculating each triangle.
they are just meant for production.
what you need is a lightweight 3D model like STEP, or even OBJ (quadrilateral mesh), FBX (with materials) - the latter are supported by three.js

**Comment by danlohman** [comment]
No problem.  If the file format can be loaded in three.js, then I totally agree that the smaller the file, the better.  Can you direct me to the most current version of the complete station in Fusion360?  If there isn't a 3D model of a completely  assembled station, no worries, we can work from the hardware component list being created to guide the gathering of the 3D models.

**Comment by Sky360Admin** [comment]
there is a complete assembly (still without the Pi5 - because we're not 100% settled on it) and this assembly contains tons of referenced components. if you export an assembly to a mesh, you end up with a huge single mesh, you can't work with. so, it's better to export every component into FBX - which is certainly a month work.

**Comment by realsnick** [comment]
is there a way to automate it?
